### PR TITLE
fix: better descriptions and bug fixes

### PR DESCRIPTION
**Pull request by StrangeGirlMurph**

### CODE REVIEW

File: src/commands/ActivityTrackerCommand.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ class ActivityTrackerCommand extends Command {
 				await statsAll(interaction);
 			}
 		} else if (group === "admin") {
-			if (await !hasAdminRole(interaction)) {
+			if (!hasAdminRole(interaction)) {
```
Comment (Chicken) [inline src/commands/ActivityTrackerCommand.ts:45]: ??

File: src/util/country-info/countryDataLoader.ts [hunk 2]
```diff
@@ -9,17 +9,27 @@ export async function initializeCountryData() {
...
-				logger.debug(err);
-				throw err;
-			})
-			.then((res) => {
-				return res as CountryImport[];
-			})
-	).sort((a, b) => b.population - a.population);
+	const response = await fetch(url)
+		.catch((err) => {
+			throw err;
```
Comment (Chicken) [inline src/util/country-info/countryDataLoader.ts:14]: maybe this should also just log and error and peacefully return

**Review by Chicken — CHANGES_REQUESTED** [review]

File: src/commands/ActivityTrackerCommand.ts (hunk 1 above)
Comment (StrangeGirlMurph) [reply src/commands/ActivityTrackerCommand.ts:45]: vscode told me it wasn't necessary. I think a promise in an if statement always gets awaited (?)

Comment (Chicken) [reply src/commands/ActivityTrackerCommand.ts:45]: It is necessary, the old code was wrong too, the `!` turned the promise into a boolean and that couldn't be awaited. This one is correct as it's first awaited and then inverted.

Comment (StrangeGirlMurph) [reply src/commands/ActivityTrackerCommand.ts:45]: oh okay yeah

File: src/util/country-info/countryDataLoader.ts (hunk 2 above)
Comment (Chicken) [reply src/util/country-info/countryDataLoader.ts]: Right, the catch needs to be moved to be after the `.then()` too

**Review by Chicken — APPROVED** [review]

**Pull request merged**